### PR TITLE
Dropzone image upload demo

### PR DESCRIPTION
**Pull request by yemexx1**
Add demo for Dropzone Image Upload Widget and Yii2 Object Storage component

### CODE REVIEW

**Comment by kenny-wealth** [comment]
Please resolve conflicts

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,9 @@
         "vlucas/phpdotenv": "^2.2",
         "lukasoppermann/http-status": "^2.0",
         "ogheo/yii2-htmlcompress": "^1.1",
-        "cottacush/yii2-utils": "~1.7.0"
+        "cottacush/yii2-utils": "dev-features/file-upload-service",
```
Comment (olajideoye) [inline composer.json]: @yemexx1  The composer versions here should be reverted to stable versions

File: composer.json [hunk 2]
```diff
@@ -13,7 +13,9 @@
...
+        "cottacush/yii2-widgets": "dev-master",
```
Comment (olajideoye) [inline composer.json:17]: @yemexx1 The composer versions here should be reverted to stable versions

**Review by olajideoye — CHANGES_REQUESTED** [review]
The versions for some of our composer dependencies need to be reverted back to stable versions.

**Pull request closed without merging**